parse_date.re

```diff
@@ -166,6 +166,8 @@ static const timelib_tz_lookup_table timelib_timezone_utc[] = {
 	{ "utc", 0, 0, "UTC" },
 };
 
+#define MAX_ABBR_LEN 5
```

Comment:
Where is this constant coming from? tzdata abbreviations [can have 3 to 6 characters, inclusive](https://data.iana.org/time-zones/data/theory.html#abbreviations).

Formally, this limitation comes from compliance with the [POSIX format for `TZ` environment strings](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html#:~:text=std%20and%20dst), and must be between 3 and `TZNAME_MAX`, inclusive.

Reply:
timelib doesn't define abbreviations longer than 5 characters. See [timezonemap.h](https://github.com/derickr/timelib/blob/master/timezonemap.h) and [fallbackmap.h](https://github.com/derickr/timelib/blob/master/fallbackmap.h). It also uses single character abbreviations.

OK. Lets change this to "6". This will help anyway.